<?xml version="1.0"?>
<launch>
  
  <param name="bound_a" value="2.0" />
  <param name="safe_bar" value="0.1" />
  <param name="kinova_max_vel" value="0.3" />
  <param name="kinova_max_a" value="1e9" />
  <param name="res_distance" value="0.2" />
  <param name="dmin" value="0.3" />
  <param name="balance_index" value="0.8" />

  <param name="robot_description" command="$(find xacro)/xacro.py '$(find kinova_description)/urdf/j2n6s300_standalone.xacro'" />
  
  <node name="fake_joint_trajectory_action_server" pkg="kinova_driver" type="fake_joint_trajectory_action_server" output="screen" respawn="false">

  </node>

  <node name="fake_joint_trajectory_cotroller" pkg="kinova_driver" type="kinova_fake_joint_trajectory_controller" output="screen">

  </node>

  <node name="qp_solver" pkg="kinova_planning" type="qp_solver.py" output="screen">

  </node>


</launch>